best-practices/native/nativescript/evironment.md

```diff
@@ -0,0 +1,80 @@
+The best practice recommends not to use more than 2 properties, `production` and `apiUrl`.
+To achieve that, we need to make some changes on environment files as you can see below:
+```
+https://medium.com/most-wanted-experts/angular-application-configurations-best-practices-2e28514b9b4e
+```
+
+The configuration on `angular.json` does not make effect when we buld the application in NativeScript. Below we have a work around to use environment.
+```
+https://medium.com/@derekfong/nativescript-angular-setup-environment-ts-for-different-environments-631b4c5219b6
+```
+
+1 - This goes to `environment.<dev|stage|prod>.ts`
+```typescript
+export const environment = {
+    production: true,
+    apiUrl: 'http://www.yoursite.com'
+};
+```
+
+2 - On environment.tns.ts:
+```typescript
+import { environment as devEnvironment } from './environment.dev';
+import { environment as prodEnvironment } from './environment.prod';
+// TODO: import additional environment (e.g. uat) if required.
```

Comment:
What does `uat` stand for?

Reply:
UAT and SIT are the two different levels of testing in the application testing phase of QA. UAT stands for User Acceptance Testing and SIT stand for System Integration Testing.

I replace UAT to stage, because we make some reference to stage later in the code, but not UAT.